service/src/integrationTest/kotlin/fi/espoo/evaka/application/ApplicationStateServiceIntegrationTests.kt

```diff
@@ -161,11 +177,58 @@ class ApplicationStateServiceIntegrationTests : FullApplicationTest() {
             // when
             service.sendApplication(tx, serviceWorker, applicationId)
         }
-        db.read {
-            // then
-            val application = fetchApplicationDetails(it.handle, applicationId)!!
-            assertEquals(LocalDate.now().plusWeeks(2), application.dueDate)
+        // then
+        assertDueDate(applicationId, null) // missing attachment
+
+        // when
+        assertTrue(uploadAttachment(applicationId, AuthenticatedUser.Citizen(testAdult_1.id)))
+        db.transaction { tx ->
+            tx.createUpdate("UPDATE attachment set received_at = :receivedAt")
```

Comment:
This updates all the attachments. Tests should not trust on existing data, but still it would be nice to work only on the specific data the tests actually uses

Reply:
True that. We are also resetting the database in `@BeforeEach`, so there should not be any other rows that get updated here. I had a plan to support `received_at` as a parameter in the attachment upload endpoint, but it wouldn't be used in production code (yet), so I refrained from doing so.